src/Gridwich.SagaParticipants.Publication.MediaServicesV3/tests/MediaServicesV3PublicationServiceTests.cs

```diff
@@ -41,8 +41,8 @@ public class MediaServicesV3PublicationServiceTests
                 new StreamingPath(StreamingPolicyStreamingProtocol.Hls, EncryptionScheme.CommonEncryptionCbcs, new List<string>() { "https://test.azure.net/ffhg/test.ism/manifest(format=m3u8)" })
             });
 
-        private static readonly StreamingEndpoint StreamingEndpoint1 = new StreamingEndpoint(name: "default", hostName: "test.azure.net", scaleUnits: 0);
-        private static readonly StreamingEndpoint StreamingEndpoint2 = new StreamingEndpoint(name: "customname", hostName: "testcustom.azure.net", scaleUnits: 1);
+        private static readonly StreamingEndpoint StreamingEndpoint1 = new StreamingEndpoint(location: "westus", name: "default", hostName: "test.azure.net", scaleUnits: 0);
```

Comment:
Why location is hard coded "westus"?

Reply:
The AMS v6 SDK requests a location parameters when creating a streaming endpoint. It is hard coded but has no impact as this is for testing and the resources is not created in real.